packages/components/src/UserCard/UserCard.tsx

```diff
@@ -103,6 +103,10 @@ export type UserCardProps = {
     flag?: string;
     phone?: string;
     profileUrl?: string;
+    link?: {
```

Comment:
Ich find es komisch mit dem default. Wollen wir nicht lieber den Link teil aus dem user objekt rausziehen und dann kann man das ja in der component default setzen?
```
user: { ... },
link: {target?: string, title: string}     // title bitte nicht optional!
 
const CompactUserCard: React.FC<Pick<UserCardProps, 'user'>> = ({
  user,
  link: {'title': 'User card', 'target': 'blanl'}
  children,
}) => {	}
```

Reply:
Und bitte nicht wieder in einem Objekt verschachteln 🙈 target sollte alle Werte nehmen, die das HTML-Attribut auch nimmt, und `_blank` als default gesetzt haben.

Reply:
Aber da ja profileUrl eigentlich dazugehört, müsste das dann ja mit raus aus dem user Objekt, oder nicht?